.github/workflows/ci.yml

```diff
@@ -181,13 +196,14 @@ jobs:
           ignore-unfixed: 'true'
       - name: Upload Trivy scan results to GitHub Security tab
         uses: github/codeql-action/upload-sarif@v1
+        if: ${{ !startsWith(github.ref, 'refs/heads/release') }}
```

Comment:
Why not upload the scan results on the release branch? Or all the other artifacts.

Reply:
I'm blocking the uploads on the release branches just to err on the side of being less invasive until the team has a chance to decide.  @pleshakov @f5yacobucci @kate-osborn What do you guys think?  I'm also blocking the upload of the unit test coverage report.

Reply:
I'm just assuming we will want to use release branches the same way we're using them in the service mesh and kic but I could change that.  @f5yacobucci @kate-osborn @pleshakov What do you think?

Reply:
(discussed in a meeting)

> Why not upload the scan results on the release branch? Or all the other artifacts.

👍 

> Is it the plan to create release branches? It might be overkill at this point in the project 🤔

we'd like to use release branches for flexibility around releases - ability to add stuff to a branch before a release and also do patch releases.